package.json

```diff
@@ -65,6 +65,7 @@
     "bugs": {
         "url": "https://github.com/hexalang/hexa-vscode-bundle/issues"
     },
-    "dependencies": {
+    "devDependencies": {
+        "vsce": "^2.9.1"
```

Comment:
Can you create a completely separate `package.json` for `vsce`? Cause this one is used by the extension itself @acidicMercury8

Reply:
Update: check my comment about npx